src/api/console_stringifier.py

```diff
@@ -33,16 +34,61 @@ def _modify_unit(self, unit: str) -> str:
         """
         Returns the modified unit.
         """
-        unit = (
-            unit.replace(r"\squared", "^2")
-            .replace(r"\cubed", "^3")
-            .replace("\\per\\", "/")
-            .replace(r"\per", "/")
-            .replace("\\", " ")
-            .strip()
-        )
-
-        if unit[0] == "/":
-            unit = f"1{unit}"
+
+        # Detect "\squared" etc.:
+        unit = unit.replace(r"\squared", "^2").replace(r"\cubed", "^3")
+        unit = re.sub(r"(\s+)\^", "^", unit)
```

Comment:
I didn't understand the purpose of this line. Do you intend to `strip()` the passed in string?

Reply:
Kind of. The idea is that units and their multiplicity (e.g. `^2`) are treated as one unit in the next processing steps.

Without this line:
`\meter \squared` → `meter ^2`

With this line:
`\meter \squared` → `meter^2`

However, I do not want to get rid of all empty spaces – just those in front of the character `^`.

Reply:
In the long-term, we probably not only want to remove whitespaces here, but everywhere. So, I've done that in 64b7d84d